db/custom_seeds/notifications.rb

```diff
@@ -0,0 +1,396 @@
+# Notification seed file, to mock notifications for a specific user (ideally yourself).
+# To run the command, run:
+# rails db:seed:notifications USERNAME='your_username'
+# you can also add CLEAN_SETUP="y" if you want to start with fresh notifications.
```

Comment:
I wish there was a cleaner way to do this -- basically I have to pass in _any_ argument in order for the ENV variable to be `present` :/